docs/quickstart.md

```diff
@@ -0,0 +1,164 @@
+<!--[metadata]>
++++
+title = "Quickstart Guide"
+description = "Getting started with Docker Compose"
+keywords = ["documentation, docs,  docker, compose, orchestration, containers"]
+[menu.main]
+parent="smn_workw_compose"
+weight=3
++++
+<![end-metadata]-->
+
+
+## Quick start
+
+Let's get started with a walkthrough of getting a simple Python web app running
+on Compose. It assumes a little knowledge of Python, but the concepts
+demonstrated here should be understandable even if you're not familiar with
+Python.
```

Comment:
I took a quick glance over this QuickStart, and I think it's safe to say that *no* knowledge of Python is required here. Each step is explained and copy-paste able :smile:

Mary is a lot better at this than I am, but a quick shot:
```Markdown
This quick start demonstrates how to run a simple web app using Docker Compose.
The web app in this tutorial is written in Python but the same concepts are applicable
to other languages, and no knowledge of Python is required.
```